fnet/cli/init.py

```diff
@@ -69,12 +73,12 @@ def save_default_train_options(path_save: str) -> None:
         'interval_save': 1000,
         'iter_checkpoint': [],
         'n_iter': 250000,
-        'path_save_dir': dirname,
+        'path_save_dir': str(path_save.parent),
         'seed': None,
     }
-    with open(path_save, 'w') as fo:
+    with path_save.open('w') as fo:
```

Comment:
`pathlib.Path` works with the default `open` function, but TIL `Path` objects have an `open` function too.